resources/templates/darcula/terminal.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/** @var $this \CaptainSpain\WebTerminal\Terminal */
+
+function makeInputLine($data, $fromData = true)
```

Comment:
This function is exactly the same as in `templates/default.php`, can you move this function to `helpers.php` or create a seperate file for functions that are being used in templates.